lib/package.json

```diff
@@ -9,20 +9,24 @@
     "directory": "lib"
   },
   "scripts": {
-    "build": "npm-run-all -p lint compile-ts compile-scss copy-scss copy-package-json copy-assets",
+    "build": "npm-run-all output-frontend output-server",
```

Comment:
The basic idea here is that we create a new output folder called `dist/server` which includes another copy of all of the RSC output files, but compiled in such a way that they are compatible with next.js.  At this point the only difference between the two is that the `server` build uses commonjs imports instead of ES6.

Reply:
> The basic idea here is that we create a new output folder called dist/server which includes another copy of all of the RSC output files, but compiled in such a way that they are compatible with next.js. At this point the only difference between the two is that the server build uses commonjs imports instead of ES6.

Actually, this has all been removed as I realized that with the next plugin we're using it's unnecessary.  Next can run on the exact same build output as frontend usage.